include/manif/impl/sgal3/SGal3_base.h

```diff
@@ -92,14 +92,14 @@ struct SGal3Base : LieGroupBase<_Derived> {
 
   /**
    * @brief Get the action
-   * @param[in] v A 3D point.
+   * @param[in] p A 3D point.
    * @param[out] -optional- J_vout_m The Jacobian of the new object wrt this.
    * @param[out] -optional- J_vout_v The Jacobian of the new object wrt input object.
```

Comment:
J_pout m
J_pout_p

Reply:
Done.

Reply:
J_pout_p
J_pout_m

see the `pout` instead of `vout`